src/Dockerfile

```diff
@@ -1,65 +1,86 @@
 FROM docker.io/library/centos:7@sha256:285bc3161133ec01d8ca8680cd746eecbfdbc1faa6313bd863151c4b26d7e5a5
 
 ENV VERSION=%%VERSION%% \
-    HELM_VERSION=%%HELM_VERSION%% \
+    HELM2_VERSION=%%HELM2_VERSION%% \
+    HELM3_VERSION=%%HELM3_VERSION%% \
     KUSTOMIZE_VERSION=%%KUSTOMIZE_VERSION%% \
+    SEISO_VERSION=%%SEISO_VERSION%% \
     IMAGE_CLEANUP_VERSION=%%IMAGE_CLEANUP_VERSION%% \
     KUBEVAL_VERSION=%%KUBEVAL_VERSION%% \
     ARCHIVE=%%ARCHIVE%% \
     SHA256SUM=%%SHA256SUM%% \
-    HELM_SHA256SUM=%%HELM_SHA256SUM%% \
+    HELM2_SHA256SUM=%%HELM2_SHA256SUM%% \
+    HELM3_SHA256SUM=%%HELM3_SHA256SUM%% \
     KUSTOMIZE_SHA256SUM=%%KUSTOMIZE_SHA256SUM%% \
+    SEISO_SHA256SUM=%%SEISO_SHA256SUM%% \
     IMAGE_CLEANUP_SHA256SUM=%%IMAGE_CLEANUP_SHA256SUM%% \
     KUBEVAL_SHA256SUM=%%KUBEVAL_SHA256SUM%% \
     SOPS_VERSION=%%SOPS_VERSION%% \
     SOPS_RELEASES_URL="https://github.com/mozilla/sops/releases/download" \
     OKD_RELEASES_URL="https://github.com/openshift/origin/releases/download" \
-    HELM_RELEASES_URL="https://storage.googleapis.com/kubernetes-helm" \
+    HELM_RELEASES_URL="https://get.helm.sh" \
     KUSTOMIZE_RELEASES_URL="https://github.com/kubernetes-sigs/kustomize/releases/download" \
-    IMAGE_CLEANUP_RELEASES_URL="https://github.com/appuio/image-cleanup/releases/download" \
+    SEISO_RELEASES_URL="https://github.com/appuio/seiso/releases/download" \
```

Comment:
The repository URL changed, and hosts "both" projects, the old and the new name. :heavy_check_mark: